department-of-veterans-affairs/abd-vro
# Remove unneeded reference to spock

Since there are no tests yet in the project, remove this unneeded spock reference.

## What was the problem?
Unneeded reference in build.gradle

Associated tickets or Slack threads:

- [Dependabot PR 541](https://github.com/department-of-veterans-affairs/abd-vro/pull/541)

## How does this fix it?[^1]
Removes reference from build.gradle

## How to test this PR
- Test SecRel [^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
